src/menus/HelpMenus.cpp

```diff
@@ -574,9 +580,8 @@ BaseItemSharedPtr HelpMenu()
 #else
       ,
 #endif
-
          // DA: Does not fully support update checking.
-   #ifndef EXPERIMENTAL_DA
+   #if !EXPERIMENTAL_DA && defined(HAVE_UPDATES_CHECK)
```

Comment:
Please be consistent with other tests of EXPERIMENTAL flags:  check whether defined,
don't check the value.
```
#if !defined(EXPERIMENTAL_DA) && defined(HAVE_UPDATES_CHECK)
```